### PR TITLE
Conceptual content edits across new nav pages

### PR DESCRIPTION
**Pull request by quaid**
This is a group PR of all my edits to the content behind the new nav buttons.

I'll be adding these pages eventually to this PR:

- Our Purpose
- Community
- Docs & Training
- Op1st Community Cloud
- Getting Started

### CODE REVIEW

File: src/components/nav-tabs/AboutPage.js [hunk 1]
```diff
@@ -8,31 +8,61 @@ export function AboutContent() {
         <Container pb={205}>
             <Title order={2} my="md">Our Purpose</Title>
             <Text pb="sm">
-              Open source software is widely available, but it faces an
-              operatations barrier to entry into production environments.
+              Why build an all-Open Source community cloud? How does it help learn
+              things and make things better?
```
Comment (durandom) [inline src/components/nav-tabs/AboutPage.js:12]: learn *stuff* ?

File: src/components/nav-tabs/AboutPage.js [hunk 2]
```diff
@@ -8,31 +8,61 @@ export function AboutContent() {
...
+              developers.
             </Text>
             <Text pb="sm">
-              To overcome this operations barrier in an open source way, we
-              must switch to an <b><i>open source approach to operations</i></b>.
-              This means that developers and operators collaborate to apply a
+              To overcome this operations barrier in an Open Source way, we
+              must switch to an <b>Open Source-like approach to operations</b>,
+              or <i>Open Operations</i> (<i>Open Ops</i>). Open Operations means
+              developers and operators collaborating Openly to apply a
```
Comment (durandom) [inline src/components/nav-tabs/AboutPage.js:26]: Do we also want to include `Open Source Services` and link to https://www.operate-first.cloud/community/open-source-services.html ?

**Review by tumido — APPROVED** [review]
/approve
/lgtm

**Comment by tumido** [comment]
Squash merging since Tide is on PTO.

**Pull request merged**